openreview/conference/templates/authorWebfield.js

```diff
@@ -113,21 +114,14 @@ function renderContent(authorNotes, invitations, tagInvitations) {
   $(tasksOptions.container).empty();
 
   Webfield.ui.newTaskList(invitations, tagInvitations, tasksOptions);
-  $('.tabs-container a[href="#author-tasks"]').parent().show();
 
   // Your Private Versions and Your Anonymous Versions tabs
-  if (authorNotes.length) {
-    Webfield.ui.submissionList(authorNotes, {
-      heading: null,
-      container: '#your-submissions',
-      search: { enabled: false },
-      displayOptions: paperDisplayOptions
-    });
-
-    $('.tabs-container a[href="#your-submissions"]').parent().show();
-  } else {
-    $('.tabs-container a[href="#your-submissions"]').parent().hide();
-  }
+  Webfield.ui.submissionList(authorNotes, {
```

Comment:
Were you able to figure out why the other tabs had a big block of empty space at the top?

Reply:
Nop, I decided to keep the same order and set the submission tab active as default.

Reply:
OK